shotover-proxy/example-configs/redis-cluster-tls/docker-compose-with-key.yml

```diff
@@ -0,0 +1,101 @@
+version: '3.3'
+services:
+  redis-node-0:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    ports:
+      - "2220:6379"
+    volumes:
+      - redis-cluster_data-0:/bitnami/redis/data
+      - ../redis-tls/certs:/usr/local/etc/redis/certs
+
+    environment: &node_environment
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+      - 'REDIS_TLS_PORT=6379'
+      - 'REDIS_TLS_ENABLED=true'
+      - 'REDIS_TLS_CERT_FILE=/usr/local/etc/redis/certs/redis.crt'
+      - 'REDIS_TLS_KEY_FILE=/usr/local/etc/redis/certs/redis.key'
+      - 'REDIS_TLS_CA_FILE=/usr/local/etc/redis/certs/ca.crt'
+      - 'REDIS_TLS_AUTH_CLIENTS=yes'
+
+  redis-node-1:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    ports:
+      - "2221:6379"
+    volumes:
+      - redis-cluster_data-1:/bitnami/redis/data
+      - ./../redis-tls/certs:/usr/local/etc/redis/certs
+    environment: *node_environment
+
+  redis-node-2:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    ports:
+      - "2222:6379"
+    volumes:
+      - redis-cluster_data-2:/bitnami/redis/data
+      - ./../redis-tls/certs:/usr/local/etc/redis/certs
+    environment: *node_environment
+
+  redis-node-3:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    ports:
+      - "2223:6379"
+    volumes:
+      - redis-cluster_data-3:/bitnami/redis/data
+      - ./../redis-tls/certs:/usr/local/etc/redis/certs
+    environment: *node_environment
+
+  redis-node-4:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    ports:
+      - "2224:6379"
+    volumes:
+      - redis-cluster_data-4:/bitnami/redis/data
+      - ./../redis-tls/certs:/usr/local/etc/redis/certs
+    environment: *node_environment
+
+  redis-node-5:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    ports:
+      - "2225:6379"
+    volumes:
+      - redis-cluster_data-5:/bitnami/redis/data
+      - ./../redis-tls/certs:/usr/local/etc/redis/certs
+    environment: *node_environment
+
+  redis-cluster-init:
+    image: docker.io/bitnami/redis-cluster:6.0-debian-10
+    depends_on:
+      - redis-node-0
+      - redis-node-1
+      - redis-node-2
+      - redis-node-3
+      - redis-node-4
+      - redis-node-5
+    volumes:
+      - ./../redis-tls/certs:/usr/local/etc/redis/certs
+    environment:
+      - 'REDIS_CLUSTER_REPLICAS=1'
+      - 'REDIS_CLUSTER_CREATOR=yes'
+      - 'ALLOW_EMPTY_PASSWORD=yes'
+      - 'REDIS_NODES=redis-node-0 redis-node-1 redis-node-2 redis-node-3 redis-node-4 redis-node-5'
+      - 'REDIS_TLS_PORT=6379'
+      - 'REDIS_TLS_ENABLED=true'
+      - 'REDIS_TLS_CERT_FILE=/usr/local/etc/redis/certs/redis.crt'
+      - 'REDIS_TLS_KEY_FILE=/usr/local/etc/redis/certs/redis.key'
+      - 'REDIS_TLS_CA_FILE=/usr/local/etc/redis/certs/ca.crt'
+      - 'REDIS_TLS_AUTH_CLIENTS=false'
```

Comment:
Why is this here?

Reply:
Are you talking about `- 'REDIS_TLS_AUTH_CLIENTS=false'`?

Reply:
yeah